podcasts/DiscoverViewController.swift

```diff
@@ -47,9 +47,9 @@ class DiscoverViewController: PCViewController {
 
         title = L10n.discover
 
-        setupSearchBar()
         handleThemeChanged()
         reloadData()
+        setupSearchBar()
```

Comment:
Had to move this below because adding the search controller as a child wasn't working when the call was before.

No idea why. 🙃